src/registries/menus/table_style_menu_registry.ts

```diff
@@ -0,0 +1,26 @@
+import { Action, createActions } from "../../actions/action";
+import { _t } from "../../translation";
+import { SpreadsheetChildEnv } from "./../../types/env";
+
+export function createTableStyleContextMenuActions(
```

Comment:
From a functional POV, there's no hint that this context menu exists and you can delete the custom table from here

Reply:
I didn't know excel so it didn't work for me. But I don't have any good idea either